docs/source/deployment/knative.rst

```diff
@@ -1,34 +1,36 @@
-Deploying to KNative
+Deploying to Knative
 ====================
 
-Knative is kubernetes based platform to deploy and manage serverless workloads. It is a
+Knative components build on top of Kubernetes to simplify deploying and managing serverless workloads. It is a
```

Comment:
should it be phrased like "Knative is a component that build ..."

Reply:
I see you already merged it :) thanks. But just to clarify, no - Knative is a set of components, namely Serving, Eventing, and an optional `kn` CLI, that all offer different features to achieve this simplification / management.